packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -555,6 +556,11 @@ class _CupertinoNavigationBarState extends State<CupertinoNavigationBar> {
 /// user scrolls, but it will also stretch when the user over-scrolls if the
 /// [stretch] value is `true`. Defaults to `false`.
 ///
+/// The [expandedTransparent] parameter determines whether the background
+/// should be transparent when expanded. The background will be transparent
+/// if the [expandedTransparent] value is true, otherwise it will take
+/// the [backgroundColor]. Defaults to `false`.
```

Comment:
For Cupertino widgets, in most cases we want the default, out of the box behavior to match their native equivalent, so I think it would be best for this to default to true. I could see wanting to avoid a sudden change in how widgets appear, but this is a smaller difference, and developers are usually appreciative of their widgets appearing more native after updating their Flutter version. And if not, they could still opt out.